### PR TITLE
Filter options

### PR DESCRIPTION
**Pull request by aginsberg**

### CODE REVIEW

File: src/ui/components/filters/filteroptionscomponent.js [hunk 1]
```diff
@@ -68,6 +68,13 @@ export default class FilterOptionsComponent extends Component {
      */
     this._onChange = config.onChange || function () {};
 
+    /**
+     * The label to br used in the legend
```
Comment (Delerme) [inline src/ui/components/filters/filteroptionscomponent.js]: typo

**Review by Delerme — CHANGES_REQUESTED** [review]
any template changes needed?

File: src/ui/components/filters/filteroptionscomponent.js (hunk 1 above)
Comment (aginsberg) [reply src/ui/components/filters/filteroptionscomponent.js]: oops, will fix
I can include the template change in this cr as well

File: src/ui/templates/controls/multioption.hbs [hunk 2]
```diff
@@ -1,10 +1,21 @@
...
-    value="{{value}}"
-    data-index="{{@index}}"
-    {{#if checked}}checked{{/if}}>
-  <label for="js-yext-checkbox-{{../name}}-{{@index}}">{{label}}</label>
-{{/each}}
+<fieldset class="Multioption-fieldset">
+  <legend class="Multioption-legend">
+    {{#if label}}
+      {{label}}
+    {{else}}
```
Comment (Delerme) [inline src/ui/templates/controls/multioption.hbs]: I'd probably remove the if/else logic if label is always expected to be defined

**Review by Delerme — CHANGES_REQUESTED** [review]

File: src/ui/templates/controls/multioption.hbs (hunk 2 above)
Comment (aginsberg) [reply src/ui/templates/controls/multioption.hbs]: Good point, will remove

**Comment by Delerme** [comment]
lgtm, to billy

File: src/ui/components/filters/filteroptionscomponent.js [hunk 3]
```diff
@@ -20,6 +20,7 @@ const SUPPORTED_CONTROLS = [
 export default class FilterOptionsComponent extends Component {
   constructor (config = {}) {
     super(config);
+    console.log(config)
```
Comment (b1lly) [inline src/ui/components/filters/filteroptionscomponent.js]: woops

**Review by b1lly — CHANGES_REQUESTED** [review]

File: src/ui/components/filters/filteroptionscomponent.js (hunk 3 above)
Comment (aginsberg) [reply src/ui/components/filters/filteroptionscomponent.js]: This has been removed!

**Review by b1lly — APPROVED** [review]
nice job.

**Pull request merged**